schema.sql

```diff
@@ -1,14 +1,55 @@
-/* Database schema to keep the structure of entire database. */
+/* Database schema to keep the structure of the entire database. */
 
-CREATE TABLE animals
-(  
-    id int,
-    name varchar(100),
+CREATE TABLE animals (
+    id serial PRIMARY KEY,
+    name varchar(255),
     date_of_birth date,
-    escape_attempts INT,
+    escape_attempts integer,
     neutered boolean,
-    weight_kg decimal
+    weight_kg decimal(10, 2)
 );
 
+ALTER TABLE animals ADD COLUMN species varchar(255);
+
+
+CREATE TABLE owners (
+    id serial PRIMARY KEY,
+    full_name varchar(255),
+    age integer
+);
+CREATE TABLE species (
+    id serial PRIMARY KEY,
+    name varchar(255)
+);
 ALTER TABLE animals
-ADD species VARCHAR(100);
\ No newline at end of file
+ALTER COLUMN id ADD GENERATED ALWAYS AS IDENTITY;
+ALTER TABLE animals
```

Comment:
- [ ] Please double-check your work before asking for a review. I checked your queries and it seems like you have some issues while altering the table `animals`. Kindly look at the below error and fix it to have error-free queries.

![image](https://github.com/firstpj/vet-clinic/assets/66476329/45986568-614e-4f09-bb76-783c64d29701)

Reply:
- [ ] As mentioned by the previous reviewer, Kindly fix this error before submitting again